src/proxy/outbound.rs

```diff
@@ -165,8 +165,8 @@ impl OutboundConnection {
                 metrics::SecurityPolicy::unknown
             },
             destination_service: None,
```

Comment:
I think we want this one as well?

Reply:
I think `destination_service` is the hostname in L7,  In ztunnel, we can't get the hostname info of L7?

Reply:
We can - its already in the Service object.

We get a service VIP, and lookup the `Service` from the VIP. This has name, hostname, namespace.

Its not the `Host` header of the request we are reporting (well, not directly -- often it is the same)

Reply:
I see. Changed to use `Arc` to reference `Service`, and set `destination_service`, `destination_service_name`, `destination_service_namespece` with the `hostname`, `name` and `namespace` of `Service`.